packages/client/src/components/Outcome/OutcomeContent.tsx

```diff
@@ -35,12 +35,22 @@ const OutcomeContent: FunctionComponent<OutcomeContentProps> = ({
   outcomeType,
   showDiscaimer,
 }) => {
+  const slug = useSlug();
   const { isPermitCheck, isPermitForm } = useTopic();
+  const { topicData } = useTopicData();
   const { matomoTrackEvent } = useTracking();
   const { t } = useTranslation();
 
   const { footerContent, mainContent, title } = outcomeContent;
 
+  // When the PreQuestion `MultipleCheckers` has been answered with `true` we will show an alternate text above the `NewCheckerModal`
+  const multipleCheckersText =
+    t(
+      `outcome.${slug}.you would like to build more than 1 so you need to do multiple permits`
+    ) +
+    " " +
```

Comment:
Is this prettier that styles it like this?

Reply:
This is Prettier, yes